Tests/AppRemovalWorkflow.Tests.ps1

```
@@ -0,0 +1,178 @@
+BeforeAll {
+    function GetTargetUserForAppRemoval { 'AllUsers' }
+    function GetInstalledAppsViaWinget { param($TimeOut, [switch]$NonBlocking) @() }
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'TimeOut' has been declared but not used.

(PSReviewUnusedParameter)